app/controllers/api/v1/meetings_controller.rb

```diff
@@ -27,7 +27,12 @@ def start
           return render_error status: :bad_request unless e.key == 'idNotUnique'
         end
 
-        render_data data: BigBlueButtonApi.new.join_meeting(room: @room, name: current_user.name, role: 'Moderator'), status: :created
+        render_data data: BigBlueButtonApi.new.join_meeting(
+          room: @room,
+          name: current_user.name,
```

Comment:
Can we extend the `join_meeting` interface (method signature) so that like with the room argument takes in a user argument and extract anything it needs from the record itself.
That will remove the need to have some logic done in/pre  the method call like `current_user.avatar.attached? ? url_for(current_user.avatar) : nil` and gives more flexibility in the future and stability to the interface,
Something like `join_meeting(room:, user: role:)` maybe?

Reply:
Tried that at first and couldn't get it in a clean way. It's tricky because the user is optional, meaning that you have to have these conditionals either way

Reply:
I'm going to refactor some of this in the near future to clean up complexity and duplication